src/servers/middlewares/auth-router.ts

```diff
@@ -215,8 +227,13 @@ authRouter.post(
           result: authToken,
         })
       } catch (err) {
-        recordExceptionInCurrentSpan({ error: err })
-        return res.status(500).send({ error: `${err.message}` })
+        if (err instanceof Error) {
+          recordExceptionInCurrentSpan({ error: err })
```

Comment:
```suggestion
          recordExceptionInCurrentSpan({ error })
```
maybe lets rename err to error so we don't need this `error: err`?